### PR TITLE
HOTFIX: Fixes == to is_a? for Integer.

### PR DESCRIPTION
**Pull request by ashercodeorg**
Fixes breakage caused by #12675.

NOTE: This is opened against test.

### CODE REVIEW

**Comment by aoby** [comment]
LGTM. Make sure you verify it works and forms can be submitted.

File: pegasus/helpers/form_helpers.rb [hunk 1]
```diff
@@ -49,7 +49,7 @@ def nil_if_empty(value)
 
 def required(value)
   return value if value.class == FieldError
-  return value if value.class == Integer
+  return value if value.class.is_a? Integer
```
Comment (bcjordan) [inline pegasus/helpers/form_helpers.rb]: Should it just be `value.is_a?`

Comment (ashercodeorg) [reply pegasus/helpers/form_helpers.rb]: Thanks!

Comment (aoby) [reply pegasus/helpers/form_helpers.rb]: Yes! Can't believe I didn't notice that :(

**Pull request merged**